superset/connectors/sqla/views.py

```diff
@@ -481,9 +481,9 @@ def _delete(self, pk: int) -> None:
 
     @expose("/edit/<pk>", methods=["GET", "POST"])
     @has_access
-    def edit(self, pk: int) -> FlaskResponse:
```

Comment:
The type passed is actually `str` (discovered whilst testing a custom override).